Added my contribution on the contibutors list

mohdnadeem330 opened the pull request:
Thank You for this Wonderfull tutorial Rahul, It helped me a lot and boosted my confidence in learning git and lwc

Commit:
Added my contribution on the contibutors list

rahulmalhotra commented:
Thank you for your PR @mohdnadeem330, glad you liked the tutorial 😄 Can you remove `config.json` from commit and update the PR? Everyone will have a different devhub so this should not be committed.

mohdnadeem330 committed:
Delete config.json

Thank you Rahul for your comment. If possible then please make tutorial on GitHub actions and how to configure CI to deploy the changes to target org, I will really appreciate your help and effort...

mohdnadeem330 commented:
Hi Rahul, I have deleted the config.json from the pull request.

rahulmalhotra commented:
Thank you for the update. It looks perfect! Merging it right now.
